### DISCUSSION TITLE
Deeper into the word probabilities

### DISCUSSION THREAD

**Discussion by mehdigreefhorst** [deepgram/community#921] [General help] [answered]
Hi Deepgram & the community,

I am working with Deepgram to construct subtitles. I want to built a system that works with the probability of the predicted words in the transcript. I wonder if it is also possible to view the other words that the system has considered. 

Thank you for your time and I'm looking forward for your response,

Mehdi Greefhorst

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by mehdigreefhorst** [reply]
Thank you for your response.

My question is not related to a specific request. Below is an example: The confidence for each word is provided in the transcript returned by Deepgram. For example, in the word below, the system predicted the word "swarmageddon" with a confidence of 0.5896303. I wonder if it is possible to retrieve the alternative words that Deepgram's AI considered to be the word with lower confidences.


{
                                "word": "swarmageddon",
                                "start": 239.875,
                                "end": 240.375,
                                "confidence": 0.5896303,
                                "punctuated_word": "Swarmageddon,"
                            },
                            {
                                "word": "as",
                                "start": 241.315,
                                "end": 241.55501,
                                "confidence": 0.99865365,
                                "punctuated_word": "as"
                            },

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @mehdigreefhorst, historically we had a feature called `Alternatives`, where you could request multiple alternative transcripts with varying probabilities. That vestige remains in our JSON response structure (`results.channels.alternatives`). However, we no longer support that feature, because we found that it ultimately didn't have much value for customers, and involves running transcription multiple times so that it requires multiple times more resources to provide those results.

Our transcription is not deterministic, so you could transcribe the same audio file multiple times, but you will always get only the highest-probability response.
